src/protocol/Connection.h

```diff
@@ -125,7 +125,9 @@ class Connection : public QObject
      */
     enum class Purpose {
         Unknown,
```

Comment:
It seems a little weird to have unknown as the first value of the enum. But not important.